test/integration/commands/xmlupload/stash/test_upload_stash_with_mock.py

```diff
@@ -99,6 +99,33 @@ def test_upload_link_value_stash(self) -> None:
         _upload_stash(upload_state, ProjectClientStub(con, "1234", None))
         assert not upload_state.pending_stash or upload_state.pending_stash.is_empty()
 
+    def test_upload_link_value_stash_multiple(self) -> None:
```

Comment:
Could you make a test where it fails to resolve the IRI. If you do the user warning you can also test that the function emitted a warning.

Reply:
hm,  I would suggest, let's discuss it for another PR, I wouldn't write that now, because it doesn't affect the problem with the omitted stash items.